### ISSUE TITLE
Not converting digits after decimal place

### ISSUE THREAD

**Issue by manishgupta33366** [allegro/tradukisto#34]
Example: Try converting 42.22
it's giving output as: czterdzieści dwa PLN 22/100
instead of:  Czterdzieści dwa złote dwadzieścia dwa grosze (converted using https://math.tools/calculator/currency/words in the Polish language)

**Comment by nilaytiwari** [comment]
in which branch this issue is coming.

**Comment by wjur** [comment]
I agree that it generates different output than you expect, but the output is correct, @manishgupta33366. This is how we write numbers in invoices in Poland. The translation you gave is correct, but it is not how the number would be written in an invoice.

If you take a look how other languages are handled in Tradukisto you will see that the same applies to other languages. Some of them even skip spaces (see: [ItalianValuesTest](https://github.com/allegro/tradukisto/blob/master/src/test/groovy/pl/allegro/finance/tradukisto/internal/languages/italian/ItalianValuesTest.groovy)).

The main purpose of Tradukisto was to convert money to text representation. This is why Tradukisto can handle only values with no more than two digits after the decimal point. 

We are open to any suggestions and feature requests. Let's discuss it and see what can be done to satisfy your requirement. What do you use Tradukisto for?

**Comment by MaciejLipinski** [comment]
As @wjur said, this is a design choice. We're open to discussing the addition of separate converters which would work as described in this issue.
